src/components/feed/post-minified.tsx

```diff
@@ -0,0 +1,30 @@
+// Modules
+import { format } from "date-fns";
+
+// Components
+import Title from "@components/global/title";
+import { PostTitle } from "@components/feed/post";
+
+// Types
+import { MinifiedPost as MinifiedPostType } from "@utils/types/post";
+
+const MinifiedPost = ({ post }: { post: MinifiedPostType }) => {
+  return (
+      <li className="hover:bg-light-primary-0.08-tlc">
+        <button className="card container-surface text-left">
+          <Title
+            icon={
+              <div className="grid place-items-center w-10 h-10
+                text-light-on-secondary font-medium bg-light-secondary rounded-full">
+                {post.author[0]}
```

Comment:
Can you trust that the backend will send back a username with at least 1 letter?